src/bot/integrations/discord.ts

```diff
@@ -371,6 +402,42 @@ class Discord extends Integration {
     } catch (e) {
       warning(e.stack);
     }
+
+    this.changeClientOnlinePresence();
```

Comment:
can you move this to first line? (to be consistent with updateStreamStartAnnounce())

Reply:
Changed with [729bcf5](https://github.com/liadala/sogeBot/commit/729bcf5447f1bc815ddd566aeeaa192117855f40)